scss/atoms/typography/_typography.scss

```diff
@@ -124,5 +132,5 @@ code {
 	background: #f2f2f2; // TODO replace with color variable
 	font-size: em( 16px );
 	padding: em( 4px, 16px ) em( 8px, 16px );
-	border-radius: 3px;
+	border-radius: 3px; // TODO replace with variable
```

Comment:
more there should be a variable (since the border radius for <code> is pretty opinionated)